### PR TITLE
Var and cvar feedback

### PR DESCRIPTION
**Pull request by ssanderson**
References from PR review point here.

### CODE REVIEW

File: zipline/finance/performance/tracker.py [hunk 1]
```diff
@@ -523,12 +517,31 @@ def _calculate_rolling_expected_shortfall(self, data_portal):
...
-            assets=assets,
+            assets=list(weights.columns),
             benchmark=self.benchmark_asset,
             data_portal=data_portal,
             end_date=sim_params.end_session,
             lookback_days=len(sim_params.sessions) + days_before_start,
         )
+        asset_returns_values = asset_returns.values
+
+        def rolling_shortfall():
```
Comment (ssanderson) [inline zipline/finance/performance/tracker.py]: If we decide to use this, I'd probably break this out into a function that takes `asset_returns`, `weights`, and `cvar_cutoff` as parameters.

File: zipline/finance/performance/tracker.py [hunk 2]
```diff
@@ -523,12 +517,31 @@ def _calculate_rolling_expected_shortfall(self, data_portal):
...
+            cvar_cutoff = zp.DEFAULT_CVAR_CUTOFF
+            out = np.full(len(weights), np.nan)
+
+            # Compute from back to front, since that simplifies the task of
+            # aligning the correct row of `weights with the correct slice of
+            # `asset_returns`.
+            last_end = -len(weights)
+            end = -1
+            while end >= last_end:
+                # TODO: Bail if we don't have enough input data.
```
Comment (dmichalowicz) [inline zipline/finance/performance/tracker.py]: I don't think we want to bail, we want to just compress the window and use what we have, down to a certain point.

**Comment by dmichalowicz** [comment]
@ssanderson I'm finally getting back to this. I think the stuff you have here is good, so to make it easier to work with I'm just going to merge this into my PR now if that's ok.

**Comment by ssanderson** [comment]
@dmichalowicz sounds good to me.

**Pull request merged**